src/test/java/com/akademia/projectplanner/validation/TaskValidatorTest.java

```diff
@@ -0,0 +1,67 @@
+package com.akademia.projectplanner.validation;
+
+import com.akademia.projectplanner.dto.TaskDto;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.time.LocalDate;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class TaskValidatorTest {
+
+  private TaskDto taskDto;
+
+  @BeforeEach
+  void beforeEach() {
+    taskDto = new TaskDto();
+  }
+
+  @Test
+  void testTaskNonBlank() {
+    taskDto.setStatus("In progress");
+    taskDto.setName("Task number 1");
+    assertFalse(TaskValidator.hasBlankNameOrStatus(taskDto));
+  }
+
+  @Test
+  void testTaskBlankNameAndStatus() {
+    taskDto.setStatus("");
+    taskDto.setName("");
+    assertTrue(TaskValidator.hasBlankNameOrStatus(taskDto));
+  }
+
+  @Test
+  void testTaskBlankStatusNonBlankName() {
+    taskDto.setStatus("");
+    taskDto.setName("Test task");
+    assertTrue(TaskValidator.hasBlankNameOrStatus(taskDto));
+  }
+
+  @Test
+  void testTaskBlankNameNonBlankStatus() {
+    taskDto.setStatus("In progress");
```

Comment:
a jeśli byśmy dostali np: "In regress"  to przejdzie - wydaje mi się, że trzeba by te statusy do enuma wyciągnąć ale to możemy przegadać jutro jeszcze jak to by zrobić